ethstore/src/error.rs

```diff
@@ -32,12 +33,13 @@ pub enum Error {
 impl fmt::Display for Error {
 	fn fmt(&self, f: &mut fmt::Formatter) -> Result<(), fmt::Error> {
 		let s = match *self {
-			Error::Io(ref err) => format!("{}", err),
+			Error::Io(ref err) => err.to_string(),
 			Error::InvalidPassword => "Invalid password".into(),
 			Error::InvalidSecret => "Invalid secret".into(),
 			Error::InvalidAccount => "Invalid account".into(),
+			Error::InvalidKeyFile(ref reason) => format!("Invalid key file: {}", reason),
```

Comment:
that's not a good plan. should print a loud warning with an explanation if why if couldn't import, but since import is an implicit action (user didn't ask for it) it should not kill the client.